tests/net/ipv6/src/main.c

```diff
@@ -1113,6 +1113,96 @@ static void test_dad_timeout(void)
 #endif
 }
 
+#define NET_UDP_HDR(pkt)  ((struct net_udp_hdr *)(net_pkt_udp_data(pkt)))
+
+static void setup_ipv6_udp(struct net_pkt *pkt,
+			   struct in6_addr *remote_addr,
+			   struct in6_addr *local_addr,
+			   u16_t remote_port,
+			   u16_t local_port)
+{
+	static char payload[] = { 'f', 'o', 'o', 'b', 'a', 'r', '\0' };
```

Comment:
What's wrong with:
~~~
static char payload[] = "foobar";
~~~

?

Or you want 2 trailing zeros? Well, "foobar\0" still works.

Reply:
This is just an old function copied from another test so not sure why it was done like that. I can send new version that simplifies this part.

Reply:
Better to adjust commit message as was suggested ;-)